src/main/model/graph.ts

```diff
@@ -73,6 +74,12 @@ export class Graph implements t.Graph {
         const i = this.nodes.findIndex(_ => _.id === nodeId);
         if (i > -1) {
             this.nodes.splice(i, 1);
+            for (const elem of Object.keys(this.refs)) {
```

Comment:
Tiny: the more conventional local variable name for keys is `key` — or something more specific like it could be `uri` in our case.

Same with array items: should be `item` if it's general-purpose. `elem` can be quite misleading in some other contexts like frontends.